locator/tablets.hh

```diff
@@ -185,6 +185,10 @@ enum class tablet_transition_kind {
     // The new replica is (tablet_transition_info::next - tablet_info::replicas).
     // The leaving replica is (tablet_info::replicas - tablet_transition_info::next).
     rebuild,
+
+    // Tablet replica is either added or removed.
+    // The new replica is the same as the one it clones, except it's located on a different node/shard.
+    rf_change,
```

Comment:
I still think this new transition kind should _not_ be introduced. As Tomek [wrote](https://github.com/scylladb/scylladb/pull/16723#issuecomment-2129735882), it was needed when the plan was to replicate data according to new RF _afterwards_ with the help of explicit rebuild/repair invocation. Now when the data is replicated automagically, we can just use the existing `transition_kind::rebuild`

Reply:
OK, removed. Replaced occurrences with rebuild.